packages/sources/peregrine-fund-admin/src/config/index.ts

```diff
@@ -0,0 +1,23 @@
+import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
+
+export const config = new AdapterConfig({
+  API_KEY: {
+    description: 'An API key for Data Provider',
+    type: 'string',
+    required: true,
+    sensitive: true,
+  },
+  API_BASE_URL: {
```

Comment:
might be what @mxiao-cll is mentioning below, but should there be default URLs here as there were in the original commit?